src/config.c

```diff
@@ -88,7 +88,12 @@ static const ConfigParam configParams[] = {
     CONFIG_PARAM(outputFile, configString, "~/Videos/ReplaySorcery_%F_%H-%M-%S.mp4"),
     CONFIG_PARAM(preOutputCommand, configString, ""),
     CONFIG_PARAM(postOutputCommand, configString,
-                 "notify-send ReplaySorcery \"Video saved!\"")};
+                 "notify-send ReplaySorcery \"Video saved!\""),
+    CONFIG_PARAM(audioDeviceName, configString, "auto"),
+    CONFIG_PARAM(audioChannels, configInt, "1"),
+    CONFIG_PARAM(audioSamplerate, configInt, "44100"),
+    CONFIG_PARAM(audioBitrate, configInt, "96000"),
+    CONFIG_PARAM(audioSystemFallback, configInt, "0")};
```

Comment:
Can you create a `configBool` which checks for the strings `true` and `false`?

Reply:
for `audioSystemCallback`, github doesn't show single-line comments very well.

Reply:
done